assets/src/BlockEditorValidation.js

```diff
@@ -10,19 +10,22 @@ let canPublish = true;
 
 export const blockEditorValidation = () => {
   subscribe(() => {
-    const blocks = select( 'core/block-editor' ).getBlocks();
+    const blocks = select('core/block-editor').getBlocks();
 
     const currentMessages = [];
-    const invalidBlocks = blocks.reduce( (invalidBlocks, block ) => {
+    // eslint-disable-next-line no-shadow
+    const invalidBlocks = blocks.reduce((invalidBlocks, block) => {
```

Comment:
What if we find another name of `invalidBlocks` of the `accumulator`, which is the `shadow` var, and avoid adding `eslint-disable-next-line no-shadow`? I know that is  not part of the scope, but the idea of using `eslint` is also to improve the quality of the code.

What do you think?

Reply:
That is a great idea, I remember not fully understanding the logic here so I didn't want to change the var name and break things, but it is a good point.